debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlPartition.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+import io.debezium.connector.common.Partition;
+import io.debezium.util.Collect;
+
+public class MySqlPartition implements Partition {
+    private static final String SERVER_PARTITION_KEY = "server";
+
+    private final String serverName;
+
+    public MySqlPartition(String serverName) {
+        this.serverName = serverName;
+    }
+
+    @Override
+    public Map<String, String> getSourcePartition() {
+        return Collect.hashMapOf(SERVER_PARTITION_KEY, serverName);
+    }
+
+    static class Provider implements Partition.Provider<MySqlPartition> {
+        private final MySqlConnectorConfig connectorConfig;
+
+        Provider(MySqlConnectorConfig connectorConfig) {
+            this.connectorConfig = connectorConfig;
+        }
+
+        @Override
+        public Set<MySqlPartition> getPartitions() {
+            return Collections.singleton(new MySqlPartition(connectorConfig.getLogicalName()));
```

Comment:
Ok, nice with the `Set`. But the `Partition` types should implement hashCode/equals now.

Reply:
Done.